lib/MusicBrainz/Server/Entity/ArtistCredit.pm

```diff
@@ -17,6 +17,23 @@ with 'MusicBrainz::Server::Entity::Role::Editable';
 
 sub entity_type { 'artist_credit' }
 
+has 'gid' => (
+    is => 'rw',
+    isa => 'Str'
+);
+
+has 'gid_redirects' => (
+    is => 'rw',
+    isa => 'ArrayRef[Str]',
+    default => sub { [] },
+    traits => [ 'Array' ],
+    handles => {
+        add_gid_redirect => 'push',
+        clear_gid_redirects => 'clear',
+        all_gid_redirects => 'elements',
+    }
+);
+
```

Comment:
Not super important, but should this be a role used by both this and CoreEntity?

Reply:
Refactored in commit https://github.com/metabrainz/musicbrainz-server/pull/2092/commits/9d0f4d1a7e6bd4ee2cf9dced1eb9324373c4e1af.